src/Renderer/QueryResultsRenderer.ts

```diff
@@ -28,6 +28,15 @@ export interface QueryRendererParameters {
     editTaskPencilClickHandler: EditButtonClickHandler;
 }
 
+function findClosestParentTask(listItem: ListItem) {
+    // Try to find the closest parent that is a task
+    let closestParentTask = listItem.parent;
+    while (closestParentTask !== null && !(closestParentTask instanceof Task)) {
```

Comment:
I expect that if we changed the `instanceof`  to something like `closestParentTask.constructor.name === 'Task'` we will remove the cyclic dependency problem (I think) we had when trying to move this code to `ListItem.ts`.